lib/application.js

```diff
@@ -202,23 +202,39 @@ class Application extends api.events.EventEmitter {
     });
   }
 
+  loadSchemas(callback) {
+    const dir = api.path.join(this.dir, 'schemas');
+    const loaderPath = api.path.join(dir, 'config.js');
+    impress.createScript(this, loaderPath, (err, getMsConfig) => {
+      if (err) {
+        callback(err);
+        return;
+      }
+
+      const { options, config } = getMsConfig();
+      options.context = this;
```

Comment:
IIUC only `sandbox` was passed previously, is this correct that we now pass whole Application?